contracts/tomox/contract/LendingRegistration.sol

```diff
@@ -111,10 +120,17 @@ contract Lending {
             require(t.issuer() == msg.sender, "Required token issuer");
         }
 
+        uint256 blockNumber = COLLATERAL_LIST[token]._blockNumber;
+        if (COLLATERAL_LIST[token]._price != price) {
```

Comment:
if externalPrice doesn't change in 2 consecutive epoch => blockNumber doesn't  change, too
In that case, tomox doesn't know whether externalPrice doesn't change or the script failed to update price to this contract